.github/wokflows/release-package.yaml

```diff
@@ -0,0 +1,31 @@
+name: mks-server image
```

Comment:
Please add licence and copyright header to this file too

Reply:
ok, done